move_base_msgs/msg/Feedback.msg

```diff
@@ -1,2 +1,3 @@
+geometry_msgs/PoseStamped goal# Goal position
```

Comment:
Why do we need this if we can read current goal from existing topic?

Reply:
You are right we already publish the current goal, but OMS needs to correlate between make_plan and feedback_navigation.Otherwise OMS has a race condition.

Reply:
Why can't it correlate with current goal published? Goal here duplicates goal from that other topic.
The goal is static for the whole duration of moving to it - this going to publish it every time.

Reply:
I understand, but OMS needs in one message.